manifests/repo.pp

```diff
@@ -32,6 +34,7 @@
       }
 
       apt::source { 'logstashforwarder':
+        ensure      => $ensure,
         location    => 'http://packages.elasticsearch.org/logstashforwarder/debian',
         release     => 'stable',
```

Comment:
does it even make sense to keep this in here given that they removed the repo and we know it doesn't work? It seems like ensure => absent is the only functionality we need.